### ISSUE TITLE
Feature Request: Allow message parts to be used in label, for example {{msg.label}} or {{msg.payload}} for example

### ISSUE THREAD

**Issue by Jibun-no-Kage** [Adorkable/node-red-contrib-ui-led#100]
Allow message parts to be used in label, for example {{msg.label}} or {{msg.payload}} for example.

**Comment by tcrussell** [comment]
Not just the label, but it would be very handy to be able to use anything as the input to the indicator itself, not just msg.payload, even flow and global variables.

**Comment by Jibun-no-Kage** [comment]
Yes, the more flexible the better, most nodes allow use of context, flow, global, for example.

**Comment by kip0x00** [comment]
> but it would be very handy to be able to use anything as the input to the indicator itself, not just msg.payload

very, very lacking!
